docs/pages/desktop-access/introduction.mdx

```diff
@@ -1,44 +1,72 @@
 ---
-title: Desktop Access
-description: Teleport desktop access introduction and resources.
+title: Manage Windows resources
+description: Demonstrates how you can manage access to Windows desktops with Teleport.
 videoBanner: n2h0GisWdss
 ---
 
-Teleport manages graphical desktop access to remote hosts. With Teleport, you
-get:
+The topics in this guide describe how to configure Teleport to provide secure, passwordless 
+access to Microsoft Windows desktops and servers. Teleport for Windows desktops provides the
```

Comment:
"Teleport for Windows desktops" kind of sounds like a separate product. I tend to prefer just "Teleport" like it was before.

Reply:
OTOH, these features are specifically features of the service for Windows rather than generic Teleport features.
For Windows, Teleport provides ....